WordPress/src/main/res/layout/new_account_user_fragment_screen.xml

```diff
@@ -87,6 +87,7 @@
                     android:layout_height="wrap_content"
                     style="@style/WordPress.NUXEditText"
                     android:inputType="textUri"
+                    android:digits="0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
That would just ignore any capital letters, which might look like a bug to the users. I think it's a better UX to just convert them automatically to lowercase.